MainRepository/Program.cs

```diff
@@ -1,7 +1,62 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿// Program to find the possible combinations of queen in the n*n chess board
+class PlaceNQueen {
+   static int Count;

```

Comment:
```suggestion
   static int sCount;

```
Static variables should start with s. Also declare this under PrintSolution. In general we have to declare these kind of variables close to the usage.